grapple/types/redirects.py

```diff
@@ -23,7 +23,7 @@ def resolve_new_url(self, info, **kwargs):
         if self.redirect_page is None:
             return self.link
 
-        return self.redirect_page.url_path
+        return self.redirect_page.url
```

Comment:
This was already updated on #109. Please merge the latest changes from the `master` branch.